lib/Payload/ZipPayload/ZipPayload.cpp

```diff
@@ -114,14 +127,17 @@ void setFilePermissions(zip_int64_t fileIndex, fs::path &fName,
                                    &attributes);
   if (opsys == ZIP_OPSYS_UNIX) {
     zip_uint32_t mask = UINT32_MAX; // all 1s for negative mask
-    mask ^= (S_IWGRP << 16);        // turn off write for the group
-    mask ^= (S_IWOTH << 16);        // turn off write for others
+    // NOLINTNEXTLINE(misc-include-cleaner)
+    mask ^= (S_IWGRP << 16); // turn off write for the group
+    // NOLINTNEXTLINE(misc-include-cleaner)
+    mask ^= (S_IWOTH << 16); // turn off write for others
```

Comment:
Why is the `misc-include-cleaner` interacting with these lines??

Reply:
The root header is in different places on different platforms.